core-plugins/src/main/java/co/cask/hydrator/plugin/batch/aggregator/AggregationUtils.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.plugin.batch.aggregator;
+
+import co.cask.cdap.api.data.schema.Schema;
+
+/**
+ * Common functions for aggregation related functionalities.
```

Comment:
The function(s) defined in this utility class seem to be unrelated to Aggregation.
They just so happen to be used in/from an aggregation plugin.

Maybe not a great name, but `NumericUtils`, `NumberUtils`, or `SchemaUtils` seems more appropriate.
What do you think?